src/beam3/4C_beam3_euler_bernoulli_evaluate.cpp

```diff
@@ -2124,7 +2124,8 @@ void Discret::Elements::Beam3eb::evaluate_stochastic_forces(
     for (unsigned int idim = 0; idim < ndim; idim++)
     {
 #ifndef BEAM3EBCONSTSTOCHFORCE
-      randnumvec(idim) = (*randomforces)(gp * randompergauss + idim).local_values_as_span()[lid()];
+      randnumvec(idim) =
+          (*randomforces).get_vector(gp * randompergauss + idim).local_values_as_span()[lid()];
```

Comment:
```suggestion
          randomforces->get_vector(gp * randompergauss + idim).local_values_as_span()[lid()];
```
... and similarly at other places.